src/customize-pop/gnome-tweaks-extensions/user-themes.md

```diff
@@ -0,0 +1,94 @@
+# Managing User Themes
+
+User themes include changes to window animations, desktop color schemes, window designs, icons, and workspaces. You can install custom GNOME Themes from opendesktop.org. The Pop!\_OS default themes can be re-applied using GNOME Tweaks or the by using Terminal commands.
+
+>**Note**: Applying custom GNOME theming options may cause system instability or glitchy behavior. Additionally, custom themes are not subject to testing by the System76 Quality Assurance team or application developers.
+
+## Enabling User Themes
+
+1. Launch the GNOME Extension Manager.
+
+    ![Launch GNOME Extension Manager](/images/user-themes/launch-ext-manager.png)
+
+2. Click `Browse` and search for `user themes`, then click `Install`.
+
+    ![Install User Themes](/images/user-themes/install-user-themes.png)
+
+3. Click `Installed` and ensure the toggle switch for `User Themes` is enabled.
+
+    ![Verify User Themes is Enabled](/images/user-themes/verify-enabled.png)
+
+## Adding a Custom User Theme
+
+> **Note**: Themes in Pop 22.04 must support GTK 4. Additionally, it may be necessary to disable [Pop!\_OS components](gnome-tweaks-extensions.md#built-in-extensions) from GNOME Extensions.
+
+### Downloading a Custom Theme
+
+Custom GNOME user themes can be downloaded from [opendesktop.org](https://www.opendesktop.org/s/Gnome/browse/).
+
+![GNOME Themes on OpenDesktop.org](/images/user-themes/open-desktop-themes.png)
+
+Download files for the theme are usually listed under `Files` section on the theme's page.
+
+### Installing a Custom Theme
+
+Follow the instructions on the theme's page to add it to your system, these are typically listed in the `Product` section on the theme's page. If no instructions are listed, click the link provided next to `Source` and view the theme's README.
+
+![Theme Installation Instructions](/images/user-themes/installation-instructions.png)
+
+Installation methods will vary, but the process will usually involve extracting a tar.gz file to either */usr/share/themes* or *~/.themes*. In the example below, a theme called `example-theme.tar.xz` is extracted to *usr/share/themes*.
```

Comment:
I would highly recommend making the example only install to `~/.themes` so you're not invoking sudo. GNOME checks both locations, so any theme should be installable to either location regardless of what its readme recommends.

Reply:
During testing I had an issue where extracting to ~/.themes didn't work, but using the other directory did work. Since this is an example I'll go with the simpler directory.